_posts/altair-to-mpl-line.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Making a Line Plot"
+date:   2018-08-15 15:30:00 -0500
+author: "Kimberly Orr and Nabarun Pal"
+categories: user-guide
+tags: "intro about line"
+excerpt_separator: <!--read more-->
+---
+
+# Making a Line Plot
+Altair works best with [long-form](https://altair-viz.github.io/user_guide/data.html#long-form-vs-wide-form-data) data. This is where each row contains a single observation along with all of its metadata stored as values.
+
+Matplotlib works a little better with wide-form data.
```

Comment:
What's the rational behind this statement?

Reply:
That was just my impression after converting so many Altair charts to Matplotlib by hand. It seems like the long-form data usually has to be reformatted into something that more resembles wide-form data for Matplotlib whenever things get slightly complicated (eg scatter plots that have categorical colors and line plots like the one in this post).

Is it incorrect to say that mpl works a little better with wide-form data? Would it be better to just leave that statement out?